include/Table.hpp

```diff
@@ -43,14 +43,48 @@ class Table {
             for(auto i = 0; i < values.size(); i++){
                 query.bind(i+1, values[i]);
             }
-            query.exec();
+            try{
+                query.exec();
+            }
+            catch(SQLite::Exception e){
+                std::string msg;
+                switch(e.getErrorCode()){
+                    case 19:
+                        msg = "Error! Unique keyword not followed!";
+                        break;
+                    default:
+                        msg = "Unknown error!";
```

Comment:
maybe print e.getErrorCode() here too

Reply:
added it to the print string, no reason to add it here